### PR TITLE
[terra-icon] Updated Devdependencies

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
This updates various devdependencies for terra-icon, and updates script syntax to reflect package changes.

### Additional Details
The following dependencies have been updated:
[csvtojson](https://www.npmjs.com/package/csvtojson) to v2.0.8
[jsdom](https://www.npmjs.com/package/jsdom) to v15.1.1
[svgo](https://www.npmjs.com/package/svgo) to v1.2.2

### CODE REVIEW

File: packages/terra-icon/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+
+### Changed
+* IconAbnormalLowLight inline style moved to scss file
 
 Unreleased
 ----------
+### Changed
+* Updated csvtojson devdependency to v2.0.8
+* Updated jsdom devdependency to v15.1.1
+* Updated svgo devdependecy to v1.2.2
```
Comment (StephenEsser) [inline packages/terra-icon/CHANGELOG.md]: A few of the entries look like they were added above the Unreleased section

File: packages/terra-icon/scripts/src/generate-example/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import parseCsv from '../icon-utils/parseCsv';
+import parseCsv from './parseCsv';
```
Comment (StephenEsser) [inline packages/terra-icon/scripts/src/generate-example/index.js]: Are the following two files identical? If so, any idea why we have two?

- ../icon-utils/parseCsv
- ./parseCsv

File: packages/terra-icon/CHANGELOG.md (hunk 1 above)
Comment (amichaelparker) [reply packages/terra-icon/CHANGELOG.md]: Updated! [95ea15f](https://github.com/cerner/terra-core/pull/2451/commits/95ea15fb56241f1bdd26df855aee6a5fd8190480)

File: packages/terra-icon/scripts/src/generate-example/index.js (hunk 2 above)
Comment (amichaelparker) [reply packages/terra-icon/scripts/src/generate-example/index.js]: They were just slightly different; one used the deprecated part of the csv, the other did not. I went ahead and updated the scripts that relied on them to only use one version, but still produce their original output: [95ea15f](https://github.com/cerner/terra-core/pull/2451/commits/95ea15fb56241f1bdd26df855aee6a5fd8190480)

File: packages/terra-icon/package.json [hunk 3]
```diff
@@ -20,11 +20,11 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
-    "csvtojson": "^1.1.4",
-    "jsdom": "^9.11.0",
+    "csvtojson": "^2.0.8",
+    "jsdom": "^15.1.1",
     "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/1.21.1.tar.gz",
```
Comment (mjhenkes) [inline packages/terra-icon/package.json]: Should we update this too? There are new icons 1.23 is out.
https://github.com/cerner/one-cerner-style-icons/releases

Comment (bjankord) [reply packages/terra-icon/package.json]: Yeah, lets go ahead and pull that in. cc/ @amichaelparker

Comment (amichaelparker) [reply packages/terra-icon/package.json]: Updated: [838e8ed](https://github.com/cerner/terra-core/pull/2451/commits/838e8ede78621db79bb557ac1d7fbc11ae47b2f2). It added some new icons and changed others, so tests and example pages were also updated.

**Pull request merged**